identity-wallet/src/state/credentials/mod.rs

```diff
@@ -95,7 +96,7 @@ impl From<CredentialFormats<WithCredential>> for VerifiableCredentialRecord {
                     metadata: CredentialMetadata {
                         is_favorite: false,
                         date_added: DateUtils::new_date_string(),
-                        date_issued: issuance_date.to_string(),
+                        date_issued: issuance_date.to_string().replace('\"', ""),
```

Comment:
If it's only about removing the encapsulating quotes, we should tackle the root of the problem and parse it to an actual `DateTime`. I think the problem stems from calling `.to_string()` on a `serde_json::Value` which wraps it in string quotes. Might be worth investigating and solve it cleanly rather than applying more bandaid here. 👍

Reply:
yes that's the problem. Changing that field to a DateTime seems like a good option. But to be consistent we would have to do that to all our timestamp fields (added, issued, first interaction, last "). I'll make it into another issue if we agree.

Reply:
https://github.com/impierce/identity-wallet/issues/204